dynamodb/dynamodb_test.go

```diff
@@ -109,48 +157,64 @@ func Test_DynamoDB_Reset(t *testing.T) {
 }
 
 func Test_DynamoDB_Close(t *testing.T) {
+	testStore, err := newTestStore(t)
+	require.NoError(t, err)
+
 	require.Nil(t, testStore.Close())
 }
 
 func Test_DynamoDB_Conn(t *testing.T) {
+	testStore, err := newTestStore(t)
+	require.NoError(t, err)
+	defer testStore.Close()
+
 	require.True(t, testStore.Conn() != nil)
 }
 
 func Benchmark_DynamoDB_Set(b *testing.B) {
 	b.ReportAllocs()
 	b.ResetTimer()
 
-	var err error
+	testStore, err := newTestStore(b)
+	require.NoError(b, err)
+	defer testStore.Close()
+
 	for i := 0; i < b.N; i++ {
 		err = testStore.Set("john", []byte("doe"), 0)
+		require.NoError(b, err)
 	}
-
-	require.NoError(b, err)
 }
 
 func Benchmark_DynamoDB_Get(b *testing.B) {
-	err := testStore.Set("john", []byte("doe"), 0)
+	testStore, err := newTestStore(b)
+	require.NoError(b, err)
+	defer testStore.Close()
+
+	err = testStore.Set("john", []byte("doe"), 0)
 	require.NoError(b, err)
 
 	b.ReportAllocs()
 	b.ResetTimer()
 
 	for i := 0; i < b.N; i++ {
 		_, err = testStore.Get("john")
+		require.NoError(b, err)
 	}
-
-	require.NoError(b, err)
 }
 
 func Benchmark_DynamoDB_SetAndDelete(b *testing.B) {
 	b.ReportAllocs()
 	b.ResetTimer()
 
-	var err error
+	testStore, err := newTestStore(b)
+	require.NoError(b, err)
+	defer testStore.Close()
+
 	for i := 0; i < b.N; i++ {
-		_ = testStore.Set("john", []byte("doe"), 0)
+		err = testStore.Set("john", []byte("doe"), 0)
+		require.NoError(b, err)
+
 		err = testStore.Delete("john")
+		require.NoError(b, err)
```

Comment:
Mmm I think this is incorrect, as it could impact the benchmarks. Changing that to have an slice of errors that will be evaluated outside of the benchmark's for loop